### PR TITLE
Adds details about TM meeting

### PR DESCRIPTION
**Pull request by bsnchan**
# Changes

Adds details about public trademark committee meeting

### CODE REVIEW

File: TRADEMARK-COMMITTEE.md [hunk 1]
```diff
@@ -58,12 +58,12 @@ subject to conformance and third-party licensing.
 
 ## Committee Meetings
 
-The committee meets as-needed. Meetings are held online.
+The trademark committee currently meets weekly, or as-needed. Meetings are open to public and held online.
```
Comment (maximilien) [inline TRADEMARK-COMMITTEE.md:61]: Why not include a link to the meeting invite here?

File: TRADEMARK-COMMITTEE.md [hunk 2]
```diff
@@ -58,12 +58,12 @@ subject to conformance and third-party licensing.
...
 
 Given the private nature of some of these discussions (e.g. trade secrets, legal
-and compliance concerns, etc.) some meetings are held in private.
+and compliance concerns, etc.) some meetings may be held in private.
 
-Meeting notes are available to members of the
+[Meeting notes](https://docs.google.com/document/d/1ghGQl0O7TM8N_YQ33w6xKyQ2yUnOX0ACsOnlqoV_XgY/edit#) are available to members of the
```
Comment (maximilien) [inline TRADEMARK-COMMITTEE.md:66]: Same thing, include link to where folks can find recordings.

**Review by maximilien — APPROVED** [review]
Thanks for adding this. Best.
/lgtm

File: TRADEMARK-COMMITTEE.md (hunk 1 above)
Comment (evankanderson) [reply TRADEMARK-COMMITTEE.md:61]: I'm guessing it's because then we need to remember to update the link here when it's updated on Google Calendar.

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

File: TRADEMARK-COMMITTEE.md (hunk 2 above)
Comment (rhuss) [reply TRADEMARK-COMMITTEE.md:66]: I would be ok to reduce the links in the markdown document to reduce the chance to become stale (as @evankanderson mentioned above). Instead add 1 canonical link with a stable URL that then contains the links to more volatile links like calendar entries or other google docs.

E.g. having a single link to the meeting notes, and the meeting notes contain the links to calendar invite + recordings, and I would mention here that additional pointers like calendar entries or the location for the recordings can be found in the document linked, too.

**Pull request merged**